cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AuthenticationUtils.java

```diff
@@ -49,7 +49,7 @@ public class AuthenticationUtils {
     /**
      * Key for OAuth config
      */
```

Comment:
Maybe you can remove the comment as well? It is not needed now and it is not really that helpful.